Blockchain/sp-oracle/script/src/index.ts

```diff
@@ -36,6 +39,12 @@ async function start(): Promise<void> {
   if (!configData) throw new Error("Input file doesn't exist");
   const config: OracleConfig = JSON.parse(configData);
   console.log('Input extracted');
+  if (config.analytics?.enabled) {
```

Comment:
There may be values ​​in the config, and they will override defaults. But when there is no input, then analytics starts by default